velox/benchmarks/basic/FormatDateTimeBenchmark.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <folly/Benchmark.h>
+#include <folly/init/Init.h>
+#include "velox/benchmarks/ExpressionBenchmarkBuilder.h"
+#include "velox/functions/prestosql/registration/RegistrationFunctions.h"
+
+using namespace facebook;
+
+using namespace facebook::velox;
+
+int main(int argc, char** argv) {
+  folly::Init init(&argc, &argv);
+  functions::prestosql::registerDateTimeFunctions("");
+
+  ExpressionBenchmarkBuilder benchmarkBuilder;
+  const vector_size_t vectorSize = 1000;
+  auto vectorMaker = benchmarkBuilder.vectorMaker();
+  auto timestampInput =
+      vectorMaker.flatVector<Timestamp>(vectorSize, [&](auto j) {
+        return Timestamp(1695859694 + j / 1000, j % 1000 * 1'000'000);
+      });
+
+  benchmarkBuilder
+      .addBenchmarkSet("", vectorMaker.rowVector({"input"}, {timestampInput}))
+      .addExpression(
+          "format_datetime",
```

Comment:
Is this a valid expression? Don't we need parentheses after function name?

Reply:
Hi @mbasmanova, this is just a name for this expression test. Maybe, no need to name it. Just removed it. Only keep the benchmark test name passed to `addBenchmarkSet`.